xbmc/music/ContextMenus.cpp

```diff
@@ -35,6 +36,22 @@
 namespace CONTEXTMENU
 {
 
+static bool IsMusicDbItem(const CFileItem& item)
+{
+  if (URIUtils::IsMusicDb(item.GetPath()))
```

Comment:
Okay, just found out that "musicdb" parent folders do havw "musicdb" URLs. I will change the code accordingly.

But I still cannot see why we always should check the node type, even if item has no music db url. Please elaboarte on this.